Gruntfile.js

```diff
@@ -1698,6 +1703,12 @@ module.exports = function(grunt) {
 		} );
 	} );
 
+	grunt.registerTask( 'wp-packages:sync-stable-blocks', 'Refresh the PHP files referring to stable @wordpress/block-library blocks.', function() {
+		grunt.log.writeln( `Syncing stable blocks from @wordpress/block-library to src/` );
+		const { main } = require( __dirname + '/tools/release/sync-stable-blocks' );
```

Comment:
Any reason we can't use?
```suggestion
		const { main } = require( './tools/release/sync-stable-blocks' );
```

Is there also any reason why it has to be required dynamically rather than in the header of the file?